metroae

```diff
@@ -1085,6 +1089,82 @@ function check_if_mount_point_configured {
     fi
 }
 
+function update_metroae {
+    debug ${FUNCNAME[0]}
+    declare -l confirmation
+    if [[ $RUN_MODE == "CONTAINER" ]]; then
+        if [[ -z $2 ]]; then
+            confirmation="init"
+            while [[ $confirmation != "n" ]] && [[ $confirmation  != "y" ]] && [[ $confirmation != "" ]]
+            do
+                echo ""
+                read -p "Do you really want to update the MetroAE Script? (y/N): " confirmation
+            done
+        else
+            confirmation=$2
+        fi
+
+        if [[ $confirmation != "y" ]]; then
+            echo ""
+            write_to_screen_and_script_log "Update of metroae Script was canceled"
+            echo ""
+            print_version_and_exit 0
+        fi
+        echo ""
+        write_to_screen_and_script_log ">>> Checking to see if we can update your metroae script."
+        echo ""
+
+        if [[ -f $SETUP_FILE ]]; then
+            while read -r line; do declare $line; done < $SETUP_FILE
+        fi
+
+        check_if_mount_point_configured
+
+        echo ""
+        write_to_screen_and_script_log ">>> Wait for the container to copy a new version file to the data directory"
+        echo ""
+        set +e
+        timeout 10 bash -c -- "while [[ ! -f $METROAE_MOUNT_POINT/version ]]; do sleep 1; done"
+        TEMP_RC=$?
+        debug $TEMP_RC
+        if [[ $TEMP_RC -ne 0 ]]; then
+            write_to_screen_and_script_log "We didn't find $METROAE_MOUNT_POINT/version."
+            echo "It could be that the version of metroae script you are running is the latest."
+            echo "Your copy of the metroae script will not be updated. You can continue to use your copy."
+            echo ""
+            set -e
+            return 1
+        fi
+        set -e
+
+        echo ""
+        write_to_screen_and_script_log ">>> Checking for a new version of the metroae script in the container data directory"
+        echo ""
+
+        if [[ ! -f "$METROAE_MOUNT_POINT/metroae" ]]; then
+            write_to_screen_and_script_log "We couldn't find a new copy of the metroae script in the data directory."
+            echo "It could be that the version of metroae script you are running is the latest."
+            echo "Your copy of the metroae script will not be updated. You can continue to use your copy."
+            echo ""
+            return 1
+        else
+            set +e
+            cp $METROAE_MOUNT_POINT/metroae "${BASH_SOURCE[0]}"
+
+            if [[ $? -ne 0 ]]; then
+                write_to_screen_and_script_log "We encountered an error copying $METROAE_MOUNT_POINT/metroae."
+                echo "Your copy of the metroae script will not be updated. You can continue to use your copy."
+                echo ""
```

Comment:
replace with

        echo "Your copy of the metroae script will not be updated. You can continue to use your copy,"
        echo "but we suggest that you update your metroae script from the Github repo before continuing."
        echo ""